src/services/apiEndpoint.spec.ts

```diff
@@ -20,7 +20,7 @@ import {
   useWsEndpoint,
 } from './apiEndpoint';
 
-function replaceEnvEach<K extends keyof typeof env>(name: K, value: (typeof env)[K]): void {
+function replaceEnvEach<K extends keyof typeof env>(name: K, value: typeof env[K]): void {
```

Comment:
Was it from Prettier? If so, I need to find a reason why it wasn't detected before.

Reply:
It wasn't from Prettier, because it has reported a problem in this PR.
```suggestion
function replaceEnvEach<K extends keyof typeof env>(name: K, value: (typeof env)[K]): void {
```

Reply:
It was formatted for me after `npm run format:all`. Should I remove it?

Reply:
Prettier says that it's invalid, so it should be rather reverted